Makefile

```diff
@@ -6,8 +6,17 @@
 PYTHON = python3
 SPHINX = sphinx-build
 BUILD_DIR = "build"
+BASE_CLASS_DIR := base_classes
+CONTRIB_DIR := contributed_definitions
+APPDEF_DIR := applications
+NYAML_SUBDIR := nyaml
 
-.PHONY: help install style autoformat test clean prepare html pdf impatient-guide all local
+YBC_NXDL = $(patsubst %.yaml,%.nxdl.xml,$(subst /nyaml/,/, $(wildcard $(BASE_CLASS_DIR)/nyaml/*.yaml)))
+YCONTRIB_NXDL = $(patsubst %.yaml,%.nxdl.xml,$(subst /nyaml/,/, $(wildcard $(CONTRIB_DIR)/nyaml/*.yaml)))
+YAPPDEF_NXDL = $(patsubst %.yaml,%.nxdl.xml,$(subst /nyaml/,/, $(wildcard $(APPDEF_DIR)/nyaml/*.yaml)))
```

Comment:
I guess YBC_NXDL, YCONTRIB_NXDL, YAPPDEF_NXDL are lists of make targets? If so please reflect that in the name.

Reply:
I added `_TARGET` as a suffix. Is this sufficient?